ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -228,11 +227,31 @@ class AlertTabs extends Component {
 
         <Tabs tabContentsClass="config-endpoint">
           <TabList customClass="config-endpoint--tabs">
-            {tabs.map((t, i) => <Tab key={tabs[i].type}>{tabs[i].type}</Tab>)}
+            {_.reduce(
+              configSections,
```

Comment:
`configSections` is a dictionary, not an array, so I don't think `.reduce` will work, no?